packages/devextreme/js/__internal/ui/chat/chat.ts

```diff
@@ -98,27 +98,52 @@ class Chat extends Widget<Properties> {
       text,
     };
 
-    // @ts-expect-error
-    this.renderMessage(message, user);
+    this.renderMessage(message);
     // @ts-expect-error
     this._messageSendAction?.({ message, event });
   }
 
+  _oneMessageWasAddedToTheEnd(value: Message[], previousValue: Message[]): boolean {
```

Comment:
It's
1. shorter (at least we should omit articles)
2. starting with a verb
```suggestion
  _isMessageAddedToEnd(value: Message[], previousValue: Message[]): boolean {
```

Reply:
[dbaeffb0202135fa22aac1fc8a6b2c01df2b7804]